docs/source/tutorials/token_classification.ipynb

```diff
@@ -231,7 +241,6 @@
    },
    "outputs": [],
    "source": [
-    "filepaths = ['data/train.txt', 'data/valid.txt', 'data/test.txt'] \n",
```

Comment:
## Executed code cell

Moved initialization of `filepaths` closer to where it is used.

This way:
- The code above can be hidden in the docs
- The subsequent code snippet is "self-contained" and can be shown in the docs.